client/my-sites/people/invite-people/index.jsx

```diff
@@ -67,6 +67,19 @@ export default React.createClass( {
 		page.back( fallback );
 	},
 
+	getTokenStatus( value ) {
+		let status;
+		if ( 'string' === typeof value ) {
+			if ( -1 < value.indexOf( 'error' ) ) {
+				status = 'is-error';
+			} else if ( -1 < value.indexOf( 'success' ) ) {
+				status = 'is-success';
+			}
+		}
+
+		return status;
+	},
```

Comment:
This implementation is temporary until we finish implementing invite validation.